pkgs/development/python-modules/openapi-core/default.nix

```
@@ -104,6 +104,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Client-side and server-side support for the OpenAPI Specification v3";
     homepage = "https://github.com/p1c2u/openapi-core";
+    changelog = "https://github.com/p1c2u/openapi-core/releases/tag/${version}"
```

```suggestion
    changelog = "https://github.com/p1c2u/openapi-core/releases/tag/${version}";
```
@fabaff That's not a changelog. Those are just release notes for the latest release.